cl/search/api_utils.py

```diff
@@ -39,19 +43,31 @@ def get_object_list(request, cd, paginator):
         "type"
     ] == SEARCH_TYPES.PEOPLE and waffle.flag_is_active(request, "p-es-active")
 
+    is_opinion_es_active = cd[
```

Comment:
Wouldn't it be more consistent to name this variable `is_opinion_active`? The other flags related to ES don't include "es" in their names.

Reply:
Sure, I've updated the variable name to `is_opinion_active`